src/app/story/storyrow/storyrow.component.html

```diff
@@ -1,6 +1,8 @@
 <div class="node-uid-column story-row-column">
   <div class="node-display-name">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
-  <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">{{ node.timeline_timestamp }}</div>
+  <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">
+    {{ node.timeline_timestamp | date: 'MMM d, y, HH:mm:ss':'+0000' }} UTC
```

Comment:
This is nit-picky for sure, but do we need a comma after the year too?

Reply:
We use commas after the year everywhere else, so I think it should be consistent. If you feel strongly about it, you can make a separate PR that fixes it everywhere.

Reply:
Nah I don't care that much, haha